cluster/aws/util.sh

```diff
@@ -1443,11 +1443,13 @@ function test-build-release {
 }
 
 # Execute prior to running tests to initialize required structure. This is
-# called from hack/e2e.go only when running -up (it is run after kube-up).
+# called from hack/e2e.go only when running -up.
 #
 # Assumed vars:
 #   Variables from config.sh
 function test-setup {
+  KUBE_TEST_DEBUG=y "${KUBE_ROOT}/cluster/kube-up.sh"
```

Comment:
setting env vars in go programs seems kinda gross... but it's probably better than duplicating this everywhere.